tests/main/hidden-snap-dir/task.yaml

```diff
@@ -184,3 +184,23 @@ execute: |
     check_env "--with-exposed-home" "x3"
     check_dirs "--with-exposed-home" "x3"
     check_data "--with-exposed-home" "x3" "$data"
+
+    echo "Check fresh install of core22 based snap"
+    snap remove --purge "$NAME"
+    rm -rf "$HOME"/Snap "$HOME"/.snap/data "$HOME/snap/$NAME"
+
+    snap install --dangerous "$NAME"_1.0_all.snap
+
+    check_env "--with-exposed-home" "x1"
+    check_dirs "--with-exposed-home" "x1"
+
+    "$NAME".cmd sh -c 'true'
+
+    echo "Check remove of freshly installed core22 based snap"
+    snap remove --purge "$NAME"
+
+    echo "Now re-install"
+    snap install --dangerous "$NAME"_1.0_all.snap
```

Comment:
do we have already a (base: core22) -> (base: core22) refresh test? otherwise we should also test that case

Reply:
That's unit tested but not spread tested AFAIK. I'll add that case then